docs/acknowledgements/development.md

```diff
@@ -0,0 +1,60 @@
+---
+id: development
+title: The Development of Metals
+---
+
+Metals is a open-source project that drew inspiration from various places
+including previous LSP servers, other ecosystems, and fresh ideas of the time.
+Metals has also been graciously given developer time and focus from
+organizations like [The Scala Center](https://scala.epfl.ch/) where the project
+originated and also companies like [Virtus Lab](https://virtuslab.com/) that now
+lead the effort in the development and maintenance of the project.
+
+## The Team
+
+While Metals has had many community contributors, here are the past and
+present maintainers of the project.
+
+_The current maintainers (people who can merge pull requests) are:_
```

Comment:
I think that @kasiaMarek and @jkciesluk are missing from this list. :wink:

Reply:
😄 ha yes I just commented on that up above. I'll add them in.